frontend/test/pytest/test_compiler.py

```diff
@@ -91,16 +92,17 @@ def test_link_lightning_runtime_input_validation(self):
         with pytest.raises(ValueError, match="is not an object file"):
             link_lightning_runtime("file-name.noo")
 
-    @pytest.mark.parametrize("verbose", [True, False])
-    def test_verbose_compilation(self, verbose, capsys):
+    @pytest.mark.parametrize("verbose,logfile", [(True, "stdout"), (True, "stderr"), (False, None)])
```

Comment:
I would prefer not to use strings here for stdout and stderr. Why not use directly sys.stdout and sys.stderr? I think one should be able to pass a file handle.

Reply:
Ah never mind, just saw how it is used in the test.